releasing/docker-run.sh

```diff
@@ -135,6 +140,7 @@ docker run --rm \
   -e GIT_VERSION="${GIT_VERSION:-$(git describe --match=v*)}" \
   -e GIT_DATE="${GIT_DATE:-$(git show -s --format=%ci)}" \
   -e GIT_HASH="${GIT_HASH:-$(git rev-parse HEAD)}" \
+  -e TARGET_LINK="${TARGET_LINK:-""}" \
```

Comment:
I think that `-e _TARGET_LINK` would also work. You don't need to specify the value if you want it to be inherited from the host.

Reply:
changed to `-e TARGET_LINK`